epitator/utils.py

```diff
@@ -56,7 +56,7 @@ def parse_number(num, default=None):
 def parse_spelled_number(num_str):
     """Parse spelled out whole numbers."""
     tokens = []
-    for t in num_str.split(' '):
+    for t in num_str.strip().split(' '):
```

Comment:
Python split with no arguments splits on all whitespace. Should we change to that?